tests/acceptance/AdminCest.php

```diff
@@ -0,0 +1,33 @@
+<?php // phpcs:disable
+
+class AdminCest {
+
+	public function moduleVersionsDisplayed( AcceptanceTester $I ) {
+		$I->wantToTest( 'About page shows module versions.' );
+		$I->loginAsAdmin();
+		$I->amOnAdminPage( 'about.php' );
+		$I->see( 'Current Module Versions' );
+		$modules = [
+			'CMS',
+			'Analytics',
+			'Cloud',
+			'Core',
+			'Developer Tools',
+			'Documentation',
+			'Search',
+			'Local Chassis',
+			'Local Server',
+			'Media',
+			'Multilingual',
+			'Privacy',
+			'Security',
+			'SEO',
+			'SSO',
+			'Workflow',
+		];
+		foreach ( $modules as $module ) {
+			$I->see( $module );
```

Comment:
This isn't checking versions right now. Maybe not the most valuable of tests so we can move on if you prefer but would be good to verify the version. Should be accessible from the `composer.lock` or `vendor/composer/installed.json`